cli/napi/js_native_api.rs

```diff
@@ -1869,7 +1867,7 @@ fn napi_get_instance_data(
   env: *mut Env,
   result: *mut *mut c_void,
 ) -> napi_status {
-  let env = &mut *(env as *mut Env);
+  let env = &mut *env;
```

Comment:
There's a few more. I think all instances of `let env = &mut *env;` are unused -- we can delete all the lines with this code.

Reply:
> There's a few more. I think all instances of `let env = &mut *env;` are unused -- we can delete all the lines with this code.

I found `let env = &mut *env;` was used in next line, like:

```rs
#[napi_sym::napi_sym]
fn napi_get_instance_data(
  env: *mut Env,
  result: *mut *mut c_void,
) -> napi_status {
  let env = &mut *env;
  let shared = env.shared(); // This line was used `env` variable
  *result = shared.instance_data;
  napi_ok
}
```

It does not make sense in this `napi_get_instance_data` function.

I'm sorry, I don't know what you mean about "we can delete all the lines with this code". Could you tell me more about that.

Reply:
Ah sorry, yeah I see where this is used now.